src/AdminPage.js

```diff
@@ -114,83 +117,83 @@ const AdminChild = () => {
   return (
     <>
       <Row>
```

Comment:
this row and column aren't needed since it lives inside of one from the App.js fire